controllers/postController.js

```diff
@@ -0,0 +1,32 @@
+const Post = require('../models/post');
+
+const add = (req, res) => {
+  const { userId,communityId,content,date,hour } = req.body
+  Post.create(userId,communityId,content,date,hour)
+}
+
+const deletePost = (req, res) => {
+  const { id } = req.params
+  Post.delete(postId)
+}
+
+const update = (req, res) => {
+  const { content } = req.body
+  const { id } = req.params
+  postId.update(content,postId)
+}
+
+const read = (req, res) => {
+  const { id } = req.params
+  Post.read(id)
```

Comment:
Functions like this that are responsible for retrieving data should be asynchronous. I'd recommend implementing a try catch block for some of these methods.